src/Core/SettingsChangesHistory.h

```diff
@@ -102,6 +102,8 @@ static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> sett
               {"function_visible_width_behavior", 0, 1, "We changed the default behavior of `visibleWidth` to be more precise"},
               {"max_estimated_execution_time", 0, 0, "Separate max_execution_time and max_estimated_execution_time"},
               {"iceberg_engine_ignore_schema_evolution", false, false, "Allow to ignore schema evolution in Iceberg table engine"},
+              {"format_template_row_format", "none", "", "Template row format string can be set directly in query"},
+              {"format_template_resultset_format", "none", "", "Template result set format string can be set in query"},
```

Comment:
```suggestion
              {"format_template_row_format", "", "", "Template row format string can be set directly in query"},
              {"format_template_resultset_format", "", "", "Template result set format string can be set in query"},
```
No need to use "none", just keep empty